src/views/view-final-confirmation.njk

```diff
@@ -0,0 +1,51 @@
+{% extends 'layout.njk' %}
+
+{% from "govuk/components/panel/macro.njk" import govukPanel %}
+{% from "govuk/components/warning-text/macro.njk" import govukWarningText %}
+
+{% block pageTitle %}
+  {{ serviceName }}
+{% endblock %}
+
+{% block content %}
+  <div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+
+      {{
+        govukPanel({
+          titleText: "Application received",
+          html: "Your reference number <br><strong>" + applicationReferenceNumber + "</strong>"
+        })
+      }}
+
+      <p class="govuk-body">We will email you within 2 working days to let you know if the application has been accepted
+        or rejected.</p>
+
+      <h3 class="govuk-heading-m">What to do next</h3>
+      <p class="govuk-body"><a href="#" target="_blank" class="govuk-link">Download the signed copy of the application
+          (PDF 238Kb) (opens in a new tab)</a>.</p>
```

Comment:
Hmmm it would be good to check with Jayti if we want to include this in this story. May raise questions in show and tell.

Reply:
Jayti is ok with that.